libraries/Parsers/Microsoft.Bot.Builder.Parsers.LU/Parser/NestedIntentSection.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.Parsers.LU.Parser
+{
+    public class NestedIntentSection : Section
+    {
+        public NestedIntentSection(LUFileParser.NestedIntentSectionContext parseTree)
```

Comment:
null validation of parse tree, check any other pre-conidtions

Reply:
Solved